examples/index.json

```diff
@@ -259,6 +266,13 @@
                 "verifyOutput": true,
                 "isLearnByExample": true
             },
+            {
```

Comment:
We can probably have a separate section on binding patterns. Along with other binding pattern BBEs like "Binding patterns in assignment". 

cc @KavinduZoysa

Reply:
Let's move the binding pattern BBEs to a separate section. Let's move https://github.com/ballerina-platform/ballerina-distribution/pull/3083/files#diff-4275ab3118a63d3c16f17a45472a9bb4150abaca5f2339463fd53c13f2f871b6R276 also.